serviceUsage.html

```diff
@@ -51,13 +51,28 @@ <h2>
             </a>Configuration
         </h2>
 
-        <p>For using Eiffel REMReM Service <strong>publish-service.war</strong> file should be deployed in Tomcat Server.
-            For doing this, publish-service.war file should locate in next directory: <em>tomcat/webapps</em>.
+        <h4>Running and configuring RemRem-Publish without Tomcat installation</h4>
+        <p>It is also possible to execute RemRem-Publish without external Tomcat installation and using the embedded tomcat in Spring application instead.</p>
+        <p>If RemRem-Publish is executed without external Tomcat installation, the configuration should be based on Spring application properties, see publish-service/src/main/resources/application.properties file for available properties.</p>
+        
+        <p>If running with java command, example:</p>
+        <pre>java -jar publish-service/target/publish-service-x.x.x.war --properties.parameter1=value --properties.parameter2=value --properties.parameter3=value ......</pre>
+        <p>It is also possible to provide path to the spring properties file:</p>
+        <pre>java -jar publish-service/target/publish-service-x.x.x.war --spring.config.location=path/to/applicattion.propeties</pre>
+
+        <p>Second option to execute RemRem-Publish is to execute maven commands from the source code root folder, example:</p>
+        <pre>mvn spring-boot:run -Dspring-boot.run.arguments=--properties.parameter1=value,--properties.parameter2=value,--properties.parameter3=value,--properties.parameter4=value ......</pre>
+        <p>When using maven command, Spring properties can also be changed by editing publish-service/src/main/resources/application.properties file.</p>
+
+        <h4>Running and configuring in Tomcat</h4>
+        <p>Eiffel REMReM Service <strong>publish-service.war</strong> file is deployed in Tomcat Server.
+            For doing this, publish-service.war file should deployed in directory: <em>tomcat/webapps</em>.
```

Comment:
I don't know how strict we are with grammar but here goes. 'For doing this' should be 'To do this' and there is also a 'be' missing here 'file should be deployed'